project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.6
+sbt.version=0.13.16
```

Comment:
We need 0.13.16 due to cross building

https://github.com/sbt/sbt/blob/v0.13.16/notes/0.13.16.markdown#sbt-cross-building